test/util/helpers.test.js

```diff
@@ -15,12 +15,19 @@ test('createGiftCardRequest', () => {
     expect(result.value.amount).toBe(20)
 })
 
-test('getEndpoint', () => {
-    const endpoint = 'EU'
-    const environment = 'production'
-    const result = getEndpoint(endpoint, endpoints, environment)
-    expect(result.location).toBe('EU')
-    expect(result.environment).toBe('production')
-    expect(result.host).toBe('agcod-v2-eu.amazon.com')
-    expect(result.region).toBe('eu-west-1')
+test('getEndpoints', () => {
+    const endpointNames = ['NA', 'EU', 'FE']
+    const environments = ['sandbox', 'production']
+    const hosts = ['agcod-v2-gamma.amazon.com', 'agcod-v2.amazon.com', 'agcod-v2-eu-gamma.amazon.com', 
+                            'agcod-v2-eu.amazon.com', 'agcod-v2-fe-gamma.amazon.com', 'agcod-v2-fe.amazon.com']
+    const regions = ['us-east-1', 'eu-west-1', 'us-west-2']
+    for (i = 0; i < 3; i++) {
+        for (j = 0; j < 2; j++) {
+            const result = getEndpoint(endpointNames[i], endpoints, environments[j])
+            expect(result.location).toBe(endpointNames[i])
+            expect(result.environment).toBe(environments[j])
+            expect(result.host).toBe(hosts[i * 2 + j])
+            expect(result.region).toBe(regions[i])
```

Comment:
Rather than having 4 expect statements, you could just say something like expect(result).toEqual(endpoints[index]), because the objects are the same.

Reply:
Done